shared/locales/date/date.en-US.properties

```diff
@@ -232,6 +232,16 @@ datePickerOrder=MDY
 # Character used in timePicker to separate hours from minutes
 hourMinutesSeparator=:
 
+# Separator used for list formatting.
+# Notice: Trailing space is meaningful - "X, Y" vs. "X,Y"
+listSeparator_middle =, 
```

Comment:
A trailing space like this won't last long (we've been there), developers love their auto-trim on save.

Use `\u0020` and add a note explaining that it represents a blank space.
```
# Separator used for list formatting. \u0020 represents a blank space
# Note that the trailing space is meaningful - "X, Y" vs. "X,Y"
listSeparator_middle =,\u0020 
```